src/main.rs

```diff
@@ -114,11 +117,13 @@ fn main() {
     fn execute_read_func_from_tuple(tin: (&str, &str)) -> ResultType {
         return match tin.0 {
             "read_list_of_lists_vect32" => {
-                ResultType::list_of_lists(read_list_of_lists_vect32(tin.1))
+                ResultType::list_of_lists(deserialize_bincoded_file(tin.1))
```

Comment:
We don't have to specify the type of `T` here, because `enum ResultType::list_of_lists` already says it